### PR TITLE
Enable pagination in demo by default, without needing to use priority policy

### PR DESCRIPTION
**Pull request by hensmi-amazon**
**Issue #:** None, but this allows coverage of pagination with 'all highest' policy

**Description of changes:** Title. Dropdown is no longer hidden behind check box. Note that the pagination buttons don't show unless there are more video sources then subscribe-able.

**Testing:**
* Tested pagination with 1 and 4 per page with and without priority policy.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
1. Switch pagination option to low value, and have more then that number of videos enabled
2. Click left and right pagination buttons.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
y

3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
n

4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
n

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: demos/browser/app/meetingV2/meetingV2.html [hunk 1]
```diff
@@ -225,15 +232,6 @@ <h5 class="modal-title" id="additional-options-modal-label">Additional Options</
             <option value="enable-bandwidth-probing">Enable Bandwidth Probing (Deprecated)</option>
             <option value="enable-bandwidth-probing-and-video-adaption" selected>Enable Bandwidth Probing and Video Quality Adaption</option>
           </select>
-          <label id="pagination-title" for="pagination-page-size" style="display: none; padding-top:5px">Paginate displayed video tiles</label>
-          <select id="pagination-page-size" class="form-select" style="width:100%; display: none;">
-            <option value="2">2</option>
-            <option value="5">5</option>
```
Comment (hensmi-amazon) [inline demos/browser/app/meetingV2/meetingV2.html:231]: In hindsight, these options feel quite silly 😅

File: demos/browser/app/meetingV2/meetingV2.html [hunk 2]
```diff
@@ -145,8 +145,7 @@ <h5 class="modal-title" id="additional-options-modal-label">Additional Options</
           </div>
           <div class="form-check" id='echo-reduction-checkbox' style="text-align: left; display: none;">
             <input type="checkbox" id="echo-reduction-capability" class="form-check-input">
-            <label for="echo-reduction-capability" class="form-check-label">Use Echo Reduction (new meetings
-              only)</label>
+            <label for="echo-reduction-capability" class="form-check-label">Use Echo Reduction</label>
```
Comment (ltrung) [inline demos/browser/app/meetingV2/meetingV2.html:148]: Thanks. I missed that when removing old Chime SDK :)

File: demos/browser/app/meetingV2/video/RemoteVideoManager.ts [hunk 3]
```diff
@@ -60,24 +65,46 @@ export default class VideoPreferenceManager {
...
+                continue;
+            }
+
+            let videoSource = new VideoSource();
+            let attendee = new Attendee();
+            attendee.attendeeId = attendeeId;
+            videoSource.attendee = attendee;
+            videoSources.push(videoSource);
+        }
+        this.logger.info(`FUCK ${JSON.stringify(videoSources)}`)
```
Comment (ltrung) [inline demos/browser/app/meetingV2/video/RemoteVideoManager.ts:106]: what is this :D

File: demos/browser/app/meetingV2/video/RemoteVideoManager.ts [hunk 4]
```diff
@@ -60,24 +65,46 @@ export default class VideoPreferenceManager {
...
+        this.downlinkPolicy.chooseRemoteVideoSources(videoSources)
```
Comment (ltrung) [inline demos/browser/app/meetingV2/video/RemoteVideoManager.ts:107]: nit: semicolon?

File: demos/browser/app/meetingV2/video/RemoteVideoManager.ts [hunk 5]
```diff
@@ -60,24 +65,46 @@ export default class VideoPreferenceManager {
...
+        for (const [attendeeId, preference] of this.attendeeIdToVideoPreference.entries()) {
+            if (this._visibleAttendees.includes(attendeeId)) {
+                videoPreferences.add(preference);
+            }
+        }
+        this.downlinkPolicy.chooseRemoteVideoSources(videoPreferences.build());
+    } else {
+        // Just convert this list to  the simpler 'VideoSource' class used by all highest policy
+        let videoSources: VideoSource[] = [];
+        for (const attendeeId of this.attendeeIdToVideoPreference.keys()) {
```
Comment (ltrung) [inline demos/browser/app/meetingV2/video/RemoteVideoManager.ts:95]: Can you remind me where we build this for non-priority downlink policy?

File: demos/browser/app/meetingV2/video/RemoteVideoManager.ts (hunk 3 above)
Comment (hensmi-amazon) [reply demos/browser/app/meetingV2/video/RemoteVideoManager.ts:106]: lol

File: demos/browser/app/meetingV2/video/RemoteVideoManager.ts (hunk 5 above)
Comment (hensmi-amazon) [reply demos/browser/app/meetingV2/video/RemoteVideoManager.ts:95]: It's this function here


```
  ensureDefaultPreferences(videoSources: VideoSource[]): void {
    this.logger.info(`Available remote video sources changed: ${JSON.stringify(videoSources)}`);
    for (const source of videoSources) {
        if (!this.attendeeIdToVideoPreference.has(source.attendee.attendeeId)) {
          this.attendeeIdToVideoPreference.set(
            source.attendee.attendeeId, 
            new VideoPreference(source.attendee.attendeeId, RemoteVideoManager.DefaultVideoTilePriority, RemoteVideoManager.DefaultVideoTileTargetDisplaySize))
        }
    }
  }
  ```

**Review by ltrung — APPROVED** [review]

**Pull request merged**